# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/dom/idlharness.any.js.ini
# Commit: d27c32ae0908
# Full Hash: d27c32ae0908c0b5d2c9213b9a624e3d0b71e2d5
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2024-01-23 21:57:50
# Regressor Bug: 1830781
# File Overlap Count: 1
# Description:
#   Bug 1830781 - Implement AbortSignal.any() r=saschanaz,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D198011
# ==============================================================================

diff -r 8af7503ecb57 -r d27c32ae0908 testing/web-platform/meta/dom/idlharness.any.js.ini
--- a/testing/web-platform/meta/dom/idlharness.any.js.ini	Tue Jan 23 08:04:52 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,28 +0,0 @@
-[idlharness.any.sharedworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [AbortSignal interface: operation any(sequence<AbortSignal>)]
-    expected: FAIL
-
-  [AbortSignal interface: calling any(sequence<AbortSignal>) on new AbortController().signal with too few arguments must throw TypeError]
-    expected: FAIL
-
-
-[idlharness.any.serviceworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [AbortSignal interface: operation any(sequence<AbortSignal>)]
-    expected: FAIL
-
-  [AbortSignal interface: calling any(sequence<AbortSignal>) on new AbortController().signal with too few arguments must throw TypeError]
-    expected: FAIL
-
-
-[idlharness.any.worker.html]
-  expected:
-    if (os == "android") and fission: [TIMEOUT, OK]
-  [AbortSignal interface: operation any(sequence<AbortSignal>)]
-    expected: FAIL
-
-  [AbortSignal interface: calling any(sequence<AbortSignal>) on new AbortController().signal with too few arguments must throw TypeError]
-    expected: FAIL